src/BloomBrowserUI/bookEdit/OverflowChecker/OverflowChecker.js

```diff
@@ -117,23 +97,22 @@ var OverflowChecker = (function () {
         else
             $page.addClass('pageOverflows');
     };
-
     OverflowChecker.CheckOnMinHeight = function (box) {
         var $box = $(box);
         var overflowy = $box.css("overflow-y");
         if (overflowy == 'hidden') {
             $box.css("min-height", $box.css("line-height"));
-        } else {
+        }
+        else {
             $box.css('min-height', '');
             var lineHeight = parseFloat($box.css("line-height"));
             var minHeight = parseFloat($box.css("min-height"));
-
             if (minHeight < lineHeight) {
                 $box.css("min-height", lineHeight + 0.01);
             }
         }
-
         $box.removeClass('Layout-Problem-Detected');
     };
     return OverflowChecker;
 })();
+//# sourceMappingURL=OverflowChecker.js.map
```

Comment:
@hatton, you can't exactly ignore the problem if git tells you there are merge conflicts in the js. However, the right action would be to do some arbitrary merge, then regenerate from the (appropriately) merged ts. That's straightforward enough that I agree we can afford to ignore the issue in code review.